README.md

```diff
@@ -404,33 +404,87 @@ With the CodePush plugin downloaded and linked, and your app asking CodePush whe
 
 2. When an update is available, how to present it to the end user?
 
-The simplest way to do this is to perform the following in your app's root component:
+The simplest way to do this is to "CodePush-ify" your app's root component. To do so, you can choose one of the following three options:
 
-1. Import the JavaScript module for CodePush:
+**Option 1: Use the [ES7 decorator](https://github.com/wycats/javascript-decorators) syntax:**
 
-    ```javascript
-    import codePush from "react-native-code-push";
-    ```
+```javascript
+import codePush, { codePushify } from "react-native-code-push";
 
-2. Call the `sync` method from within the `componentDidMount` lifecycle event, to initiate a background update on each app start:
+let codePushOptions;
 
-    ```javascript
-    codePush.sync();
+@codePushify(codePushOptions)
+class MyApp extends Component {
+}
+```
+
+*NOTE: Decorators are not yet supported in Babel 6.x pending proposal update.* You may need to enable it by doing the following:
```

Comment:
Are decorators not supported by default? If so, then perhaps it would be worth just making option 2 the default? We could maybe then also just remove this option and leave it as a footnote to option 2, only for those people who already know how to use decorators and have their project configured for it.